packages/common/src/authentication/AuthContext.tsx

```diff
@@ -10,6 +9,16 @@ import { AuthenticationResponse } from './api';
 
 export const COOKIE_LIFETIME_MINUTES = 60;
 
+type User = {
+  id: string;
+  name: string;
+};
+
+interface Store {
+  id: string;
+  code: string;
+}
+
```

Comment:
Not ideal. Though, if the types changed, it would likely be for different reasons between here and somewhere else?

Reply:
yeah, not ideal. Ok for now though - I expect the shape of these will change, and will be useful elsewhere maybe.